src/NLog/Common/InternalLogger.cs

```diff
@@ -276,6 +270,8 @@ private static void Write([CanBeNull]Exception ex, LogLevel level, string messag
                 {
                     Console.Error.WriteLine(msg);
                 }
+
+                PerformLogToDiagnostics(level, msg);
```

Comment:
Proposal: rename to "WriteToDiagnostic" - it's more in line with the current naming